financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/manage/utils/ConsentManageUtils.java

```diff
@@ -66,8 +66,8 @@ public static boolean isTransactionFromToTimeValid(String fromDateVal, String to
             OffsetDateTime fromDate = OffsetDateTime.parse(fromDateVal);
             OffsetDateTime toDate = OffsetDateTime.parse(toDateVal);
 
-            // From date is earlier than To date
-            return toDate.isAfter(fromDate);
+            // From date is equal or earlier than To date
+            return toDate.isEqual(fromDate) || toDate.isAfter(fromDate);
         } catch (DateTimeParseException e) {
             return false;
```

Comment:
Shall we add a debug log here too?

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/130/commits/1e13ec5af065388311c13d8ca7f04ef804d84184